client/src/main/java/org/eclipse/hono/client/impl/GatewayMapperImpl.java

```diff
@@ -41,9 +42,9 @@ public GatewayMapperImpl(final RegistrationClientFactory registrationClientFacto
     }
 
     @Override
-    public Future<String> getMappedGatewayDevice(final String tenantId, final String deviceId) {
+    public Future<String> getMappedGatewayDevice(final String tenantId, final String deviceId, final SpanContext context) {
         return registrationClientFactory.getOrCreateRegistrationClient(tenantId).compose(client -> {
-            return client.get(deviceId);
+            return client.get(deviceId); // TODO use SpanContext here
```

Comment:
I have merged the PR adding the *get* method variant accepting the span context. Do you want to rebase and update this right away?

Reply:
Yes, I'll do that.

Reply:
Done.